lang/csharp/build.sh

```diff
@@ -30,7 +30,7 @@ case "$1" in
 
   test)
     xbuild
-    nunit-console -framework=4.0 Avro.nunit
+    nunit-console Avro.nunit
```

Comment:
I think that this will be a problem, because the solution needs to build cleanly within the docker container out-of-the-box and this prevents it doing so. Could you remove this from this pull request, or amend something so it still works (I say "something" because I'm not sure if this requires a change to the docker setup or the solution file).